vignettes/introduction.Rmd

```diff
@@ -1,6 +1,7 @@
 ---
 title: "A. Introduction"
 output: rmarkdown::html_vignette
+runtime: shiny
```

Comment:
No obvious functionality differences on pkgdown because if you're publishing vignettes they are usually forced rendered as static HTML pages by knitr. The intention was if people did run the rmd then the document would start up the shiny runtime enabling the interactive